src/hotspot/share/gc/epsilon/epsilonHeap.cpp

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2017, 2020, Red Hat, Inc. All rights reserved.
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Here and later: please update Red Hat's copyright year instead. (Like we do for mostly-Oracle files in other cases).

Reply:
Changed as you requested.